packages/amplify-graphql-api-construct-tests/src/utils/validate-transformer-helper.ts

```diff
@@ -6,6 +6,12 @@ import { ValidationType, ValidationsByField } from '@aws-amplify/graphql-validat
 import { generateTypeValidationSnippet } from '@aws-amplify/graphql-validate-transformer/src/vtl-generator';
 import { graphqlRequest } from '../graphql-request';
 
+// ========================================================
```

Comment:
nit (future PR is fine): you can use `#region` markers to help demarcate file regions, and give you foldable areas in VSCode.